README.md

```diff
@@ -1,6 +1,23 @@
-# squiral
+![PyPI](https://img.shields.io/pypi/v/pip) [![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/) [![PyPI pyversions](https://img.shields.io/pypi/pyversions/ansicolortags.svg)](https://pypi.python.org/pypi/ansicolortags/)
```

Comment:
Actually, this project doesn't support Python 2.7.
We can get data directly from pypi, like those badges:
[ ![ ] ( https://img.shields.io/pypi/v/squiral )] (https://pypi.org/project/squiral/) 
[ ![ ] ( https://img.shields.io/pypi/pyversions/squiral.svg )] (https://pypi.org/project/squiral/)